### DISCUSSION TITLE
Multitenant?

### DISCUSSION THREAD

**Discussion by DavidDury** [mixcore/mix.core#439] [Ideas & Features]
Hi,

Is it possible to implement a multitenant solution to this? Have you taught about it?

What about v2 when do you think there will be a release?

Best Regards,
David

**Comment by Smilefounder** [comment]
Hi @DavidDury , Mixcore v1 is not fully supported multitenant yet. However, all of the thinking and planning are ready and currently, we are working on v2 at the moment. v2 will have multitenant for sure. 

> What about v2 when do you think there will be a release?
We aim to release v2 early next year. However, because of the covid situation in our country. We might delay a bit.

Best regards.

**Comment by vbasky** [comment]
Hi @Smilefounder Is multi-tenant available on the latest release ?

**Reply by Smilefounder** [reply]
This feature is not existing in v1. You can use develop branch to use this feature (Mixcore v2 only).

**Reply by vbasky** [reply]
I just checked out the develop branch. I get a bunch of .dll not found even after a `dotnet restore` I do have .net 7.0 installed. 

I remember I had to manually clone the mix.heart  repo inside platform/core as the checkout to develop coudln't build it. Let me see what the error is

**Reply by Smilefounder** [reply]
`git clone --recursive` should work. Thanks.
